### PR TITLE
Building blocks for the voice API

### PR DESCRIPTION
**Pull request by lornajane**
This is everything that made sense to me to do from curl - basically the things you can do with outgoing API calls.

### CODE REVIEW

File: voice/voice-api/play-audio-into-a-call.sh [hunk 1]
```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+source "../../config.sh"
+source "../../jwt.sh"
+
+curl -X PUT https://api.nexmo.com/v1/calls/$UUID/stream \
+  -H "Authorization: Bearer "$JWT\
+  -H "Content-Type: application/json"\
+  -d '{"stream_url": ["https://s3-eu-west-1.amazonaws.com/labio-bin/audio/welcome_to_nexmo.mp3"]}'
```
Comment (mheap) [inline voice/voice-api/play-audio-into-a-call.sh]: Should we move these to be hosted on Nexmo Developer? Like we do in `public/ncco/tts.json` for example

Comment (tbedford) [reply voice/voice-api/play-audio-into-a-call.sh]: Do you mean the MP3 files?

Comment (mheap) [reply voice/voice-api/play-audio-into-a-call.sh]: Yes, the MP3 files. At the moment this is in one of Adam's S3 buckets and could disappear at any moment

Comment (tbedford) [reply voice/voice-api/play-audio-into-a-call.sh]: OK yes, it should be moved then. Everything else looks reasonable.

Comment (lornajane) [reply voice/voice-api/play-audio-into-a-call.sh]: I might put them here https://github.com/nexmo-community/ncco-examples and tidy up that repo a bit, add a readme etc, how does that sound?

**Pull request merged**